frontend/src/components/IngredientEditor.js

```diff
@@ -1,29 +1,38 @@
-import {makeStyles, TextField} from "@material-ui/core";
+import {IconButton, makeStyles, TextField} from "@material-ui/core";
 import {useState} from "react";
 import styled from "styled-components/macro";
+import {CancelOutlined} from "@material-ui/icons";
 
-export default function IngredientEditor({ingredient}) {
+export default function IngredientEditor({ingredient, onChange, onDelete}) {
     const [ingredientData, setIngredientData] = useState(ingredient)
     const classes = useStyles();
 
-
-    const onDataChanged = nameChangedEvent => {
+    const onDataChanged = dataChangedEvent => {
```

Comment:
This looks to me like the process is happening twice. Is it possible to only use the on change function and directly do the in the RecipeDraft.js instead of first doing it locally in IngredientEditor.js ?